tests/suites.py

```diff
@@ -307,7 +307,77 @@ async def test_git_repo_conda(self) -> None:
                 path_to_source="example_package",
                 interpreter=CondaEnvironmentYmlFile("myenv.yml"),
             ),
+            True,
+        )
+
+    @pytest.mark.skipif("sys.version_info < (3, 8)")
+    @pytest.mark.parametrize(argnames="editable_install", argvalues=(True, False))
+    @pytest.mark.asyncio
+    async def test_git_repo_pip_file_with_editable_install(
```

Comment:
Smallest nit of all nits--I think this can just be named `test_git_repo_pip_with_editable_install`, will follow pattern of other test names slightly better. So sorry for even mentioning this